pkg/aws_config_client/parse_aws_config.go

```diff
@@ -29,6 +30,8 @@ type AWSOIDCConfiguration struct {
 	ClientID  string
 	IssuerURL string
 	RoleARN   string
+	Region    string
+	Output    string
```

Comment:
```suggestion
	Output    *string
```
to indicate these are optional values